src/main/java/com/gdsc/pikpet/config/security/SecurityConfig.java

```diff
@@ -0,0 +1,51 @@
+package com.gdsc.pikpet.config.security;
+
+import org.springframework.boot.autoconfigure.security.servlet.PathRequest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.crypto.factory.PasswordEncoderFactories;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+public class SecurityConfig {
+    @Bean
+    public SecurityFilterChain securityFilterChain(
+            HttpSecurity http
+    ) throws Exception {
+        http.authorizeHttpRequests(
+                authorizeHttpRequests ->
+                        authorizeHttpRequests
+                                .requestMatchers(PathRequest.toStaticResources().atCommonLocations()).permitAll()
+                                .requestMatchers("/loginPage/formLogin").permitAll()
+                                .requestMatchers("/loginPage/loginFail").permitAll()
+                                .requestMatchers("/api/**").permitAll()
```

Comment:
이거 구현시에만 열어두고 배포시에는 닫을 거니까 주석 남깁시다
```suggestion
                                .requestMatchers("/api/**").permitAll() //구현의 편의를 위해 임시 개방
```

Reply:
좋아요~~